lib/src/showcase_widget.dart

```diff
@@ -124,7 +124,7 @@ class ShowCaseWidgetState extends State<ShowCaseWidget> {
   int? activeWidgetId;
   RenderBox? rootRenderObject;
   Size? rootWidgetSize;
-  Key? anchoredOverlayKey;
+  Key anchoredOverlayKey = UniqueKey();
```

Comment:
This`anchoredOverlayKey` can be considered final as it is not updated anywhere in the code. so can you please update this?

Reply:
That is right, I have made the update.